### DISCUSSION TITLE
404 error for published Dapp

### DISCUSSION THREAD

**Discussion by nexusocean8** [WalletConnect/walletconnect-monorepo#2685] [Developer Support]
Hello, I was just wondering if it just takes time or if there's something additional I need to do to get my page setup on the explorer page.

Currently receiving 404 at this address: https://walletconnect.com/explorer/social-blocks

Thank you!

**Comment by JeroldG** [comment]
This one is now solved, we pushed a hot fix to ensure approvals are published on Explorer within 2 hours.
